index.html

```diff
@@ -0,0 +1,175 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta name="description" content="choose online tour for trip">
+  <link rel="stylesheet" href="./scss/bootstrap.scss">
+  <link rel="stylesheet" href="./styles/main.css">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Mulish:wght@400;600;700&family=Nunito:wght@400;600;800&display=swap"
+    rel="stylesheet">
+  <title>Best Tour Plan</title>
+</head>
+
+<body>
+  <header class="header">
+    <div class="service-header">
+      <div class="container">
+        <div class="row top-header">
+          <div class="head-info col-12 col-lg-6">
```

Comment:
You don't need col-12. Check all the document.

Reply:
> You don't need col-12. Check all the document.

col-12 classes removed